# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../vdr/softdevice-0.1.0-makefile.diff
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- softdevice/Makefile~	2006-04-29 08:20:28.000000000 +0200
+++ softdevice/Makefile	2006-05-02 12:43:11.000000000 +0200
@@ -25,7 +25,7 @@
 VDRDIR     = ../../..
 LIBDIR     = ../../lib
 TMPDIR     = /tmp
-PLUGINLIBDIR = ./PLUGINS/lib
+PLUGINLIBDIR = /usr/lib/vdr
 
 -include config.mak
 
@@ -81,7 +81,7 @@
 #   install -m 644 postprocess.h /usr/local/include/postproc/postprocess.h
 #   cp libpostproc.a /usr/local/lib/
 #
-PP_LIBAVCODEC=1
+#PP_LIBAVCODEC=1
 
 # -----------------------------------------------------------------------------
 # if you want output methods build as a single lib comment the following line
@@ -94,7 +94,7 @@
 #VIDIX_DIR    = /opt/vidix
 #VIDIX_DIR    = /usr/local/
 
-VIDIX_DIR    = /usr/local
+VIDIX_DIR    = /usr
 
 # -----------------------------------------------------------------------------
 # Set up these paths to your enviroment!
@@ -103,7 +103,7 @@
 #FFMPEGCFLAGS=-I../../../../ffmpeg/libavcodec
 #
 # If you installed ffmpeg's lib (make installlib)
-FFMPEGCFLAGS=-I/usr/local/include/ffmpeg/
+FFMPEGCFLAGS=-I/usr/include/ffmpeg/
 #FFMPEGCFLAGS=-I/home/extra/src/video/ffmpeg-0.4.8/libavcodec
 #
 #
@@ -143,13 +143,13 @@
 endif
 
 ifdef DFB_SUPPORT
-  DFB_LIBS  = -L/usr/local/lib -ldfb++
-  DFB_CFLAGS = -I/usr/local/include/dfb++ -I/usr/local/include/directfb
+  DFB_LIBS  = -L/usr/lib -ldfb++
+  DFB_CFLAGS = -I/usr/include/dfb++ -I/usr/include/directfb
   DEFINES  += -DDFB_SUPPORT
 endif
 
 ifdef XV_SUPPORT
-  XV_LIBS   = -L/usr/X11R6/lib -lXi -lXext -lX11 -lm -lXv
+  XV_LIBS   = -L/usr/X11R7/lib -lXi -lXext -lX11 -lm -lXv
   ifdef LIBXDPMS_SUPPORT
     DEFINES += -DLIBXDPMS_SUPPORT
   endif
